tools/changelogger/class-package-formatter.php

```diff
@@ -49,7 +49,7 @@ public function getEpilogue() {
 	 */
 	public function getReleaseLink( $version ) {
 		// Catpure anything past /woocommerce in the current working directory.
-		preg_match( '/\/woocommerce\/packages\/js\/(.+)/', getcwd(), $path );
+		preg_match( '/\/woocommerce[^\/]*\/packages\/js\/(.+)/', getcwd(), $path );
```

Comment:
I see how this is an improvement, but it raises the notion that someone could name the Monorepo whatever they'd like. Should we take it one step further and make the regex `/\/packages\/js\/(.+)/`?